build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -48,6 +54,78 @@ module.exports = function ({template, types: t}) {
     return filename.endsWith('.jss.js');
   }
 
+  function classnameId(name) {
+    return `\$${name}`;
+  }
+
+  /**
+   * @param {Path} reference
+   * @param {string} importedName
+   * @return {?Path}
+   */
+  function findImportDeclaration(reference, importedName) {
+    if (!reference.isIdentifier()) {
+      return null;
+    }
+
+    const binding = reference.scope.getBinding(reference.node.name);
+    if (!binding || binding.kind !== 'module') {
+      return null;
+    }
+
+    const {path} = binding;
+    const {parentPath} = path;
+
+    if (
+      !parentPath.isImportDeclaration() ||
+      !path.isImportSpecifier() ||
+      !t.isIdentifier(path.node.imported, {name: importedName})
+    ) {
+      return null;
+    }
+
+    return parentPath;
+  }
+
+  /**
+   * Replace a MemberExpression (like `a.x` or `useStyles().x`) with its
+   * imported constant (`$x`). It creates the necessary import as well.
+   * @param {Path} importDeclaration
+   * @param {Path} callExpression
+   * @param {Path} memberExpression
+   * @return {boolean}
+   */
+  function replaceMemberExpression(
+    importDeclaration,
+    callExpression,
+    memberExpression
+  ) {
+    const {property} = memberExpression.node;
+    if (!memberExpression.isMemberExpression() || !t.isIdentifier(property)) {
```

Comment:
Need to verify the `MemberExpression` is not `computed`.
```suggestion
    if (!memberExpression.isMemberExpression({computed: false}) || !t.isIdentifier(property)) {
```